template.tpl

```diff
@@ -441,36 +441,15 @@ ___TEMPLATE_PARAMETERS___
     "subParams": [
       {
         "type": "TEXT",
-        "name": "clientId",
-        "displayName": "Client ID",
+        "name": "conversionToken",
+        "displayName": "Conversion Token",
```

Comment:
Proposing we make the display name match the [documentation](https://business.reddithelp.com/helpcenter/s/article/conversion-access-token) so it's consistent for people searching..etc.
```suggestion
        "displayName": "Conversion Access Token",
```